python/cudf/cudf/_lib/parquet.pyx

```diff
@@ -87,3 +95,57 @@ cpdef read_parquet(filepath_or_buffer, columns=None, row_group=None,
         df.index.name = new_index_name
 
     return df
+
+cpdef write_parquet(
+        _Table table,
+        path,
+        compression=None,
+        statistics="ROWGROUP"):
+    """
+    Cython function to call into libcudf API, see `write_parquet`.
+
+    See Also
+    --------
+    cudf.io.parquet.write_parquet
+    """
+
+    # Create the write options
+    cdef string filepath = <string>str(path).encode()
+    cdef sink_info sink = sink_info(filepath)
```

Comment:
I believe the `sink_info` constructor expects a `const std::string&` so you need to get the reference to `filepath` here instead of passing it directly.